src/formatDate.js

```diff
@@ -8,7 +8,45 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[3]);
+  const objToFormat = {};
+  const objFromFormat = {};
+  const newFormat = [];
+  const getLastTwoDigits = (num) => {
+    return num % 100;
+  };
```

Comment:
The `getLastTwoDigits` function could be simplified to a one-liner without the braces and return statement. You can use an arrow function with an implicit return here.